src/__tests__/utils/test-utils.ts

```
@@ -0,0 +1,369 @@
+                    );
+                }
+                return handler(request);
+            }
+        };
+    });
+
+    return mockCheckPermission;
+};
+
+// Create a test user with specified role and permissions
+export const createTestUser = async (role: string, permissions: string[]) => {
```

ESLint

[error] 'permissions' is defined but never used.

(@typescript-eslint/no-unused-vars)